code/game/objects/structures/mirror.dm

```diff
@@ -12,14 +12,6 @@
 /obj/structure/mirror/attack_hand(mob/user as mob)

```

Comment:
While I don't support this, if you're going to axe it you might as well remove the entire attack_hand section as it exists solely for the appearance changing.